target/arm/helper.c

```diff
@@ -11093,7 +11093,7 @@ static CPUARMTBFlags rebuild_hflags_a64(CPUARMState *env, int el, int fp_el,
          * The decision of which action to take is left to a helper.
          */
         if (sctlr & (SCTLR_EnIA | SCTLR_EnIB | SCTLR_EnDA | SCTLR_EnDB)) {
-            DP_TBFLAG_A64(flags, PAUTH_ACTIVE, 1);
+            //DP_TBFLAG_A64(flags, PAUTH_ACTIVE, 1);
```

Comment:
Why?

Reply:
PAC might crash with a nullpointer called 0x02000000 or something like that otherwise.

Reply:
Eh?

Reply:
Something in Apple's IOMFB might be somehow incompatible with PAC.
It crashed way too often, so I just disabled PAC.
I'll revert it.
If it continues to prone to crash, you can disable PAC.

Reply:
I've not experienced that here, but I'm not booting the installed OS cause I've not gotten to installing iOS again yet